src/iologindata.cpp

```diff
@@ -661,7 +661,7 @@ bool IOLoginData::savePlayer(Player* player)
 	//serialize conditions
 	PropWriteStream propWriteStream;
 	for (Condition* condition : player->conditions) {
-		if (condition->isPersistent()) {
+		if (condition->isPersistent() || condition->getType() == CONDITION_MUTED) {
```

Comment:
Wouldn't the isPersistent function be a perfect place for that condition type check?

Reply:
Do you mean
```
if (!(id == CONDITIONID_DEFAULT || id == CONDITIONID_COMBAT || getType() == CONDITION_MUTED)) {
	return false;
}
```
?

Reply:
Yep.